File: WordGames/type/type.py
# A program to create a list from a file, and then create a list of all the words containing the string "type"
words = []
type = 'TYPE'
list_of_type = []

# importing the word list
sow_pods = open("/home/evergreen/UnderDogsAssignments/sowpods")

# Go through each word in the file and add it to the list, words
for line in sow_pods:
    words.append(line.strip())

# Go through the list and add it to list_of_type as appropriate
for x in range(len(words)):
    if type in words[x]:
        list_of_type.append(words[x])
        
print(list_of_type)

File: WordGames/type/type.js
// Program to find all the words containing 'type'
// Importing the fs module
let fs = require("fs");
//const { type } = require("os");

// Intitializing the readFileLines with the file
const readFileLines = filename =>
   fs.readFileSync(filename)
   .toString('UTF8')
   .split('\n');

// Calling the readFiles function with file name
let words = readFileLines('/home/evergreen/UnderDogsAssignments/sowpods');
console.log(words)

// Cycle through words and if "type" is in the index, add it to type
typeWord = []

for(let i = 0; i < words.length; i++) { 
    if (words[i].includes("TYPE")) {
        typeWord.push(words[i]);}}

console.log("All the words containing 'type' are " + typeWord)